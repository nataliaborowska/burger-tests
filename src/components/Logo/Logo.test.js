import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';

import Logo from './Logo';

it('render correctly Logo component', () => {
  const tree = renderer.create(<Logo />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('sets correct height when prop height is passed', () => {
  const props = {
    height: 50,
  };
  const LogoComponent = shallow(<Logo {...props} />);
  const logoNode = LogoComponent.find('[data-test]');

  expect(logoNode.prop('style')).toHaveProperty('height', 50);
});