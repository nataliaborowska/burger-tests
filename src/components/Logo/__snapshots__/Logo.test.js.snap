// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Logo component 1`] = `
<div
  className={undefined}
  data-test="Logo"
  style={
    Object {
      "height": undefined,
    }
  }
>
  <img
    alt="MyBurger"
    src="burger-logo.png"
  />
</div>
`;
